src/Serializers/ExpDataSerializer.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace SMW\Serializers;
+
+use Serializers\Serializer;
+use SMW\Exporter\Element\ExpElement;
+use SMWExpData as ExpData;
+use OutOfBoundsException;
+use RuntimeException;
+
+/**
+ * @license GNU GPL v2+
+ * @since 2.2
+ *
+ * @author mwjames
+ */
+class ExpDataSerializer implements Serializer {
+
+	/**
+	 * @see Serializer::serialize
+	 *
+	 * @since 2.2
+	 */
+	public function serialize( $expData ) {
+
+		if ( !$this->isSerializerFor( $expData ) ) {
+			throw new OutOfBoundsException( 'Object is not supported' );
+		}
+
+		return $this->doSerialize( $expData ) + array( 'serializer' => __CLASS__, 'version' => 0.1 );
+	}
+
+	/**
+	 * @see Serializer::isSerializerFor
+	 *
```

Comment:
`isSerializerFor` is not part of `Serializer`, so you do not need to implement it. And if you do implement it, it likely only makes sense for polymorphic usage, in which case you need to implement `DispatchableSerializer`